modules/user/user.service.js

```diff
@@ -151,11 +164,34 @@ class UserService {
       ],
     });
     const savedUser = await user.save();
+    const token = await generateToken(savedUser._id, savedUser.email);
+    const message = {
+      from: `horondi.devproject@gmail.com`,
```

Comment:
It should be taken from the environment variable

Reply:
fixed